static/Makefile

```diff
@@ -4,6 +4,8 @@ CLI_DIR=$(realpath $(CURDIR)/../src/github.com/docker/cli)
 ENGINE_DIR=$(realpath $(CURDIR)/../src/github.com/docker/docker)
 BUILDX_DIR=$(realpath $(CURDIR)/../src/github.com/docker/buildx)
 
+ENGINE_GITCOMMIT?=$(shell cd $(realpath $(CURDIR)/../src/github.com/docker/docker) && git rev-parse --short HEAD)
```

Comment:
Was wondering; is this different from `ENGINE_REF`? https://github.com/docker/docker-ce-packaging/blob/47167619d0386bee288924b58aad5f71daf1e13f/common.mk#L41

Reply:
Same as https://github.com/docker/docker-ce-packaging/blob/47167619d0386bee288924b58aad5f71daf1e13f/deb/Makefile#L6

Reply:
Ah! Yeah, we should look if we can share more of those (probably the complication was to define the correct path)